frontend/components/dashboard/table/SummaryDetails.vue

```diff
@@ -1,60 +1,77 @@
 <script setup lang="ts">
 import type { VDBSummaryTableRow } from '~/types/api/validator_dashboard'
-import { type SummaryDetailsEfficiencyCombinedProp, type SummaryRow } from '~/types/dashboard/summary'
-import { TimeFrames, type TimeFrame } from '~/types/value'
+import { type SummaryDetailsEfficiencyCombinedProp, type SummaryRow, type SummaryTableVisibility, type SummaryTimeFrame } from '~/types/dashboard/summary'
 
```

Comment:
A question regarding the mobile view:
![image](https://github.com/gobitfly/beaconchain/assets/150015231/d892b528-978c-412e-bfb1-24b159c8ca43)
when the screen width is so tight, Figma shows that that three Power buttons spread vertically:
![image](https://github.com/gobitfly/beaconchain/assets/150015231/697bb445-05bb-4f03-9820-97ed5c1555fe)

Reply:
good catch a `wrap`was missing ;-)